frontend/src/pages/article/article-page-shared.tsx

```diff
@@ -0,0 +1,59 @@
+import { type FC } from 'react';
+
+import { Layout } from '~/libs/components/components.js';
+import { getUrlLastSegment } from '~/libs/helpers/helpers.js';
+import {
+  useAppDispatch,
+  useAppSelector,
+  useEffect,
+  useLocation,
+} from '~/libs/hooks/hooks.js';
+import { type TagType } from '~/libs/types/types.js';
+import { actions as articlesActions } from '~/slices/articles/articles.js';
+
+import { ArticleView, Author } from './components/components.js';
+import styles from './styles.module.scss';
+
+const ArticlePageShared: FC = () => {
+  const dispatch = useAppDispatch();
+  const location = useLocation();
+  const article = useAppSelector(({ articles }) => {
+    const dataFromStore = articles.sharedArticle;
+    if (dataFromStore) {
+      return dataFromStore[0];
+    }
+  });
+
+  useEffect(() => {
+    const token = getUrlLastSegment(location.pathname);
+    void dispatch(articlesActions.fetchSharedArticle({ token })).unwrap();
```

Comment:
You don't use the action payload after the it has fulfilled, so I guess `.unwrap()` is unnecessary